getting-started/quickstart/README.md

```diff
@@ -0,0 +1,29 @@
+# Apache Polaris Quickstart with File-Based Backend
+
```

Comment:
Wonder whether you should just link to the markdown in `site/` to de-duplicate content. WDYT?